src/style.css

```diff
@@ -1,3 +1,62 @@
-body {
+html {
+  --header-height: 60px;
+  --text-hover-color: #00ACDC
+}
+
+* {
```

Comment:
we shouldn't reset styles like this. Move `margin: 0;` to those tags you need. 
It's allowed to use `box-sizing: border-box` at `*` selector